sdk/ml/azure-ai-ml/azure/ai/ml/_internal/entities/_input_outputs.py

```diff
@@ -106,10 +106,18 @@ def _from_base(cls, _input: Union[Input, Dict]) -> Optional["InternalInput"]:
 
 
 class InternalOutput(Output):
-    def __init__(self, *, datastore_mode=None, **kwargs):
+    def __init__(self, *, datastore_mode=None, is_link_mode=None, **kwargs):
         self.datastore_mode = datastore_mode
+        self.is_link_mode = is_link_mode
         super().__init__(**kwargs)
 
+    def _to_dict(self, remove_name=True):
```

Comment:
it's not recommended to add parameters to _to_dict. This is a function inherited from a mixin and adding params may break the inheritance.

Reply:
so what is the remove name for?

Reply:
usually you won't need to override this function if schema & related configurations are correctly set